dialoguekit/platforms/platform_terminal.py

```diff
@@ -0,0 +1,46 @@
+"""The Platform facilitates displaying of the conversation."""
+
+from typing import Callable
+
+from dialoguekit.core.utterance import Utterance
+from dialoguekit.platforms.platform import Platform
+
+
+class TerminalPlatform(Platform):
+    def __init__(self):
+        """Represents a platform."""
+        self._user_callback: Callable[[str], None] = None
+
+    def register_user_callback(self, callback: Callable[[str], None]) -> None:
+        """Registers a callback function to be called on user input.
+
+        Args:
+            callback: Function to call on user input
+        """
+        self._user_callback = callback
```

Comment:
Not sure this method is needed here as it is the same as in parent class.

Reply:
Removed.